### ISSUE TITLE
is:dupe equating adept and non-adept versions of weapons

### ISSUE THREAD

**Issue by jchapman16** [DestinyItemManager/DIM#9843]
### DIM Version

Version 7.84.0.1002892 (beta), built on 9/3/2023, 9:57:40 PM

### Browser Details

Version 116.0.5845.140 (Official Build) (x86_64)

### OS Details

MacOS 13.5 (22G74)

### Describe the bug

The is:dupe filter is equating adept and non-adept versions of weapons. For example, if you have a "Fatebringer" and a "Fatebringer (Timelost)", or a "Submission" and a "Submission (Adept)", both weapons are now being included in the filter results. Previously those two pairs of weapons would not be chosen by the filter This is a recent behavior change from all previous versions of DIM and inconsistant with how other sites (e.g. light.gg) treat them as distinct. They do have different identifiers, after all. If this isn't a bug and was an intentional change, I think there should be two different filters for the two different behaviors (e.g. "is:dupe" and "is:dupestrict").

**Comment by bhollis** [comment]
This was an intentional change in #9798. The idea is that if you have both adept and non-adept weapons, you are likely to want to consider cleaning up the non-adept versions. Can you help us understand how this change affects your workflow?

**Comment by jchapman16** [comment]
Certainly; thank you for asking!In general, I try to keep one copy of every weapon I decide to keep. I use the “is:dupe” filter extensively to support this rule; it’s my first pass to identify what can be deleted.Adept weapons from raids are the exception to the rule; the difficulty in obtaining them plus the inherent advantages in stats and mods mean that I keep all of them, along with a crafted non-adept version. This allows me to have an adept roll (usually PvE) and a crafted roll (usually PvP) for raid weapons. The past behavior of the “is:dupe” filter let me do this easily- it only returned the dupes I needed to address, not the pairs of adept and crafted rolls that I don’t consider “dupes” (since the adept weapons are inherently different from the crafted or non adept versions)I understand the point of view that drove this change; is there a combination of existing filters that can be used to keep the old “dupe” behavior? If not, can the previous dupe rules be retained in a new filter (e.g. is:dupeloose or is:dupeadept)?Thank you for responding and listening to my feedback. -ChappyOn Sep 6, 2023, at 12:44 AM, Ben Hollis ***@***.***> wrote:﻿
This was an intentional change in #9798. The idea is that if you have both adept and non-adept weapons, you are likely to want to consider cleaning up the non-adept versions. Can you help us understand how this change affects your workflow?
